PanModal/Presentable/PanModalPresentable.swift

```diff
@@ -126,6 +126,15 @@ public protocol PanModalPresentable: AnyObject {
      Default value is true.
      */
     var allowsDragToDismiss: Bool { get }
+    
+    /**
+     A flag to determine if the UIViewController should always be presented as Pan Modal.
+     
+     - Note: Return true to always be a Pan Modal
+     
+     Default is false. Which means that on iPad, it will be presented as a popover & as Pan Modal on iPhone.
+     */
+    var alwaysForcePanModal: Bool { get }
```

Comment:
I was undecided between `forcePanModal` or `alwaysForcePanModal` but went with the last to enforce that setting this to true will **always**, no matter the device, result in a PanModal